src/Migrations/M200903000000CreateHttpView.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Wearesho\Yii\Guzzle\Migrations;
+
+use yii\db;
+
+class M200903000000CreateHttpView extends db\Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp(): void
```

Comment:
In this case up/down methods have to be used. Because we are performing single query that won't break anything.